### PR TITLE
DSPDC-950 Fix SQL logic.

### PR DESCRIPTION
**Pull request by danxmoran**
Will comment explanations inline.

### CODE REVIEW

File: charts/argo-templates/scripts/query-non-null-rows.sh [hunk 1]
```diff
@@ -9,13 +9,13 @@ declare -r PK_COLS=$(cat ${TABLE_DIR}/primary-keys)
 
 # Build the WHERE clause of the SQL query.
 declare -a COMPARISONS=()
-declare -a PRIMARY_COLUMNS=()
+declare -a DATAREPO_COLUMNS=(datarepo_row_id)
 for c in ${PK_COLS//,/ }; do
   COMPARISONS+=("${c} IS NOT NULL")
-  PRIMARY_COLUMNS+=("${c} as datarepo_${c}")
```
Comment (danxmoran) [inline charts/argo-templates/scripts/query-non-null-rows.sh:15]: The `as` was a syntax error.

**Review by raaidbroad — APPROVED** [review]

File: charts/argo-templates/scripts/query-non-null-rows.sh (hunk 1 above)
Comment (quazi-h) [reply charts/argo-templates/scripts/query-non-null-rows.sh:15]: Whoops, sorry that was from my commit.
Is "col_name as alias" not valid for BQ sql syntax?

File: charts/argo-templates/scripts/query-non-null-rows.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ declare -ra BQ_QUERY=(
   --replace=true
   --destination_table=${PROJECT}:${DATASET}.${TARGET_TABLE}
 )
-1>&2  ${BQ_QUERY[@]} "SELECT * EXCEPT (datarepo_row_id, ${REPO_KEYS})
+1>&2  ${BQ_QUERY[@]} "SELECT * EXCEPT (${REPO_KEYS})
```
Comment (quazi-h) [inline charts/argo-templates/scripts/query-non-null-rows.sh:37]: "datarepo_row_id" would be returned within "REPO_KEYS", right?

File: charts/argo-templates/scripts/query-non-null-rows.sh (hunk 1 above)
Comment (danxmoran) [reply charts/argo-templates/scripts/query-non-null-rows.sh:15]: It's valid in the `SELECT` list (which is how it's used in the script we copied from), but not in the `EXCEPT` list (how it's used here).

File: charts/argo-templates/scripts/query-non-null-rows.sh (hunk 2 above)
Comment (danxmoran) [reply charts/argo-templates/scripts/query-non-null-rows.sh:37]: Yup!

**Review by snwessel — APPROVED** [review]

**Pull request merged**